packages/salesforcedx-sobjects-faux-generator/test/integration/integrationTestUtil.ts

```diff
@@ -50,6 +52,21 @@ export async function createScratchOrg(projectName: string): Promise<string> {
   return Promise.resolve(username);
 }
 
+export async function deleteScratchOrg(userName: string): Promise<void> {
```

Comment:
@chowwinston - Do we need this function? One of the new features of the CLI is the ability to create a scratch org with a "purpose". Could we change the scratch org creation to do a CLI creation purpose of CI, which would make it auto-delete in 3 days (I don't know the exact days).

Reply:
Depending on whether that can be controlled (asking platform-cli), even 3 days would be too long when testing code locally and repeatedly.  Pain in the butt to have to go to the devhub and manually remove these so frequently.

Reply:
Where are you running your tests? If you are doing this locally, you can bump up your allocation of scratch orgs. Our CI org has a high scratch org limit as well.

That being said, your point is understood. Let's just schedule it for cleaning to be good citizens.